lib/opencensus/stats/view_data.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+
+module OpenCensus
+  module Stats
+    # @private
```

Comment:
The exporter interface references this class. And the `Recorder` class also has public methods that return objects of this type. Thus, it cannot be `@private`.

Reply:
Fixed